test/specs/apitests/genre-api-test.js

```diff
@@ -0,0 +1,12 @@
+const GenresEndpoint = require("../../api/genres-endpoint");
+const ApiResponseValidator = require("../../utilities/response-validator");
+
+describe("API Tests for Genres", () => {
+  it("should return a list of genres when GET /api/genres/ is called", async () => {
+    const response = await GenresEndpoint.getAllGenres();
+
+    ApiResponseValidator.validateStatus(response);
+    ApiResponseValidator.validateArrayResponse(response);
```

Comment:
These two method calls are used in almost all API requests. Can they be called in a separate method, so that in the test, you only need one call?